### PR TITLE
Prepare for release  of sdk

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Once we are ready for the SDK release this PR should be merged.

## Type of change

- Documentation Enhancement

## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by lucas-tortora — APPROVED** [review]
LGTM, though we may want to merge https://github.com/iotaledger/iota-wiki/pull/1047 first. 

As well, we could consider keeping the old client docs and add an admonition recommending the sdk instead.

**Comment by Dr-Electron** [comment]
> LGTM, though we may want to merge #1047 first.
> 
> As well, we could consider keeping the old client docs and add an admonition recommending the sdk instead.

Yes, i need to rebase 👍🏻

And yes we would do that. @luca-moser wdyt?

**Comment by Dr-Electron** [comment]
Re-added the old docs and added deprecate admonitions:
https://iota-wiki-poqn56oxe-iota1.vercel.app/shimmer/iota.js/welcome/
https://iota-wiki-poqn56oxe-iota1.vercel.app/shimmer/iota.rs/welcome/
https://iota-wiki-poqn56oxe-iota1.vercel.app/shimmer/wallet.rs/welcome/
Also added an announcement bar: https://iota-wiki-poqn56oxe-iota1.vercel.app/shimmer/

File: shimmer/docusaurus.config.js [hunk 1]
```diff
@@ -15,6 +15,14 @@ module.exports = async () => {
     baseUrl: '/shimmer/',
     themeConfig: {
       image: 'img/shimmer-wiki.png',
+      announcementBar: {
+        id: 'iota-sdk',
+        content:
+          'The new IOTA SDK got released, <a target="_blank" href="/iota-sdk/welcome/">check it out</a>🚀. It deprecates iota.js, iota.rs and wallet.rs',
```
Comment (lucas-tortora) [inline shimmer/docusaurus.config.js:21]:
```suggestion
          'The new IOTA SDK got released, <a target="_blank" href="/shimmer/iota-sdk/welcome/">check it out</a>🚀. It deprecates iota.js, iota.rs, and wallet.rs',
```

or 

```suggestion
          '<a target="_blank" href="/shimmer/iota-sdk/welcome/">Check out the new IOTA SDK</a>. 🚀. It deprecates iota.js, iota.rs, and wallet.rs',
```

File: shimmer/docs/iota-sdk/docs/contribute.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+The project's documentation is also open source and available on GitHub. If you wish to contribute new documentation or
+fix any errors, please refer to
+the [contribution guidelines](https://github.com/iotaledger/documentation/blob/develop/.github/CONTRIBUTING.md).
+
+## Share Your Knowledge
+
+Helping others is a crucial aspect of any open source ecosystem. By sharing your knowledge with the community, you can
+provide immense value and potentially inspire others to learn and contribute. Join the ongoing discussions in the
+#clients-discussion channel on [Discord](https://discord.iota.org).
```
Comment (lucas-tortora) [inline shimmer/docs/iota-sdk/docs/contribute.md:46]: if we created a new channel we should probably update this

Comment (Dr-Electron) [reply shimmer/docs/iota-sdk/docs/contribute.md:46]:
```suggestion
#libraries channel on [Discord](https://discord.iota.org).
```

**Comment by lucas-tortora** [comment]
We should probably update https://iota-wiki-poqn56oxe-iota1.vercel.app/shimmer/develop/explanations/what-is-shimmer/clients/ as well. Do you want to do it in this PR or should I create an issue so we can do it in another?

**Comment by Dr-Electron** [comment]
> We should probably update https://iota-wiki-poqn56oxe-iota1.vercel.app/shimmer/develop/explanations/what-is-shimmer/clients/ as well. Do you want to do it in this PR, or should I create an issue so we can do it in another?

As discussed I think it might make sense to create another PR for it

> We could also add the depreciation admonition here as a quick fix and create a new article. Though I think part of the benefit of the SDK is not having to explain which you should choose.

I think we can take the time and update the article. Should be doable today and not many people will this site as their first visit anyway :trollface:

**Review by lucas-tortora — APPROVED** [review]

**Review by luca-moser — APPROVED** [review]

**Pull request merged**